### PR TITLE
[44] unit tests for contact us endpoint

### PR DESCRIPTION
**Pull request by stanislawK**
To run tests:
- get into running container: docker exec -ti codeforpoznanpl_v3_backend_1 bash
- enter pytest

### CODE REVIEW

File: backend/tests/conftest.py [hunk 1]
```diff
@@ -28,3 +29,64 @@ def app():
...
+    access_token = rv.get_json()['access_token']
+    return access_token
+
+
+@pytest.fixture
+def new_msg():
+    msg = {"name": "TestName",
+           "email": "test@test.com",
+           "phone": "777222333",
+           "content": "Lorem Ipsum cos tam cos tam"}
```
Comment (w1stler) [inline backend/tests/conftest.py]: I don't like this indentation.

File: backend/tests/test_contact_us.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+from backend.extensions import mail
+
+
+def test_contact_us_endpoint(client, new_msg):
+    """
+    GIVEN an application instance, valid message, and mocked email outbox
+    WHEN sending valid contact message to send-email endpoint
+    THEN check status code, message, and email outbox
+    """
```
Comment (w1stler) [inline backend/tests/test_contact_us.py]: As in previous PR.

File: backend/tests/test_contact_us.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from backend.extensions import mail
+
+
+def test_contact_us_endpoint(client, new_msg):
+    """Test sending valid message to send-email enndoint."""
```
Comment (w1stler) [inline backend/tests/test_contact_us.py]: endpoint

**Comment by OtisRed** [comment]
@kristhina review, please (make start, make bash, make test)

**Pull request merged**